calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -274,7 +274,9 @@ curl -O https://raw.githubusercontent.com/lsgroup/SmartReverseProxy/master/modse
 curl https://raw.githubusercontent.com/coreruleset/coreruleset/v3.3/dev/crs-setup.conf.example > crs-setup.conf
 ```
 
-3. Change your current directory to the my-ruleset folder where your core rules set files live. Create a configMap containing all the files downloaded into your new directory and replace the existing rule set with it:
+3. If you wish to customize settings on default included rules, please see `.example` files for guidance. You must save these files without the .example extension if you wish for these changes to take effect.
```

Comment:
If you wish to customize settings on default included rules, see `.example` files for guidance.